### PR TITLE
Africa module split

### PR DESCRIPTION
**Pull request by brunobord**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1 +1,2 @@
 include README.rst
+prune workalendar/tests*
```
Comment (Natim) [inline MANIFEST.in]: This is already part of #141

File: workalendar/africa/benin.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
```
Comment (Natim) [inline workalendar/africa/benin.py]: You probably do not need all of this but I like it :)

Comment (brunobord) [inline workalendar/africa/benin.py]: yes, I was thinking about dropping the "print_function", for example ;o)

File: MANIFEST.in (hunk 1 above)
Comment (brunobord) [inline MANIFEST.in]: possibly a strange rebase issue. will re-rebase or cherry-pick at some point.

**Comment by Natim** [comment]
r+

**Comment by brunobord** [comment]
it's unfinished work: I'd like to cleanup unnecessary imports (print_function, and such)

**Comment by brunobord** [comment]
Obsolete, will restart from scratch

**Pull request closed without merging**